.travis.yml

```diff
@@ -94,6 +94,12 @@ before_install:
   # - Rely on `kerl` for [pre-compiled versions available](https://docs.travis-ci.com/user/languages/erlang#Choosing-OTP-releases-to-test-against). Rely on installation path chosen by [`travis-erlang-builder`](https://github.com/travis-ci/travis-erlang-builder/blob/e6d016b1a91ca7ecac5a5a46395bde917ea13d36/bin/compile#L18).
   # - . ~/otp/18.2.1/activate && erl -version
   #- curl -f -L -o ./rebar3 https://s3.amazonaws.com/rebar3/rebar3 && chmod +x ./rebar3 && ./rebar3 version && export PATH="${TRAVIS_BUILD_DIR}:$PATH"
+  # install Qt 5.10
```

Comment:
I apologize for the late comment.
But does not it work when upstream qt distro? 
debian has `5.7.x` on `stretch` and `5.11.x` on `buster`

Reply:
`PetStore.pro` need at least `5.10` because of [QRandomGenerator](https://doc.qt.io/qt-5/qrandomgenerator.html].

How can I install `5.11.x` from `buster`?

Reply:
I create #3508 to remove this unneccessary dependency to `5.10`

Reply:
Ok good idea!

I don't understand what you mean by:

> But does not it work when upstream qt distro?
> debian has `5.7.x` on `stretch` and `5.11.x` on `buster`

What do you want me to do so we merge this PR?

Reply:
Would it be possible to check if now you can just install upstream qt with adding ppa?

Reply:
I have removed dependency to `Qt 5.10`

Reply:
Ok sorry I don't understand what you mean. I'm *MacOS* user so I'm not aware about the good practice for setting up linux qt development environment.

Reply:
ok we can merge it as it is. and improve later.